jsk_pcl_ros_utils/scripts/xyz_to_screenpoint.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import rospy
+
+from sensor_msgs.msg import CameraInfo
+from geometry_msgs.msg import PointStamped
+
+from image_geometry import PinholeCameraModel
+
+class XYZToScreenPoint:
+    def __init__(self):
+        self.cameramodels = PinholeCameraModel()
+        self.is_camera_arrived = False
+
+    def fromCameraInfo_cb(self, msg):
+        self.cameramodels.fromCameraInfo(msg)
+        self.is_camera_arrived = True
+
+    def project3dToPixel_cb(self, msg):
+        if not self.is_camera_arrived:
+            return
+        point = (msg.point.x, msg.point.y, msg.point.z)
+        x, y = self.cameramodels.project3dToPixel(point)
+        rospy.logdebug("x, y : %lf, %lf", x, y)
+        # publish info
+        pub = rospy.Publisher("~output", PointStamped, queue_size=1)
```

Comment:
this should be moved to `__init__`, otherwise publisher is created in every loop.